addons/binding/org.openhab.binding.yeelight/about_files/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (C) 2013 Q42
```

Comment:
Q42, are you sure?

Reply:
I did not write this, so no. I guess there also was a comment from @martinvw on the old PR #3255 regarding this and some other remarks from you. I thought these were already applied, but apparently not. So I will also look at them again.

Reply:
For reference: https://github.com/openhab/openhab2-addons/pull/1514#discussion_r111671913

Reply:
Am I allowed to change anything here (since I did not write most of the code)?

If yes, I'd like to apply the suggested change from https://github.com/openhab/openhab2-addons/pull/1514#discussion_r111671913
Shell I just add the link to the MIT license in the license file and remove the rest?

Reply:
Q42 is actually a company, I assume that the license had simply be copied from https://github.com/eclipse/smarthome/blob/0.7.0-release/binding/org.eclipse.smarthome.binding.hue/about_files/LICENSE.

The correct header can be found here: https://github.com/coasterli/YeelightLib/blob/master/LICENSE#L3